contracts/pallet-ibc/src/lib.rs

```diff
@@ -340,6 +340,18 @@ pub mod pallet {
 	/// Active Escrow addresses
 	pub type EscrowAddresses<T: Config> = StorageValue<_, BTreeSet<T::AccountId>, ValueQuery>;
 
+	#[pallet::storage]
+	#[allow(clippy::disallowed_types)]
+	/// Consensus heights
+	/// Stored as a tuple of (revision_number, revision_height)
+	pub type ConsensusHeights<T: Config> = StorageMap<
+		_,
+		Blake2_128Concat,
+		Vec<u8>,
+		BoundedBTreeSet<(u64, u64), frame_support::traits::ConstU32<256>>,
```

Comment:
How about using `Height` instead of `(u64, u64)` to use it's ordering implementation?